README.md

```diff
@@ -22,14 +22,17 @@ Prerequisite: [Node 20+](https://www.npmjs.com/package/node/v/20.11.1)
    ```shell
    npm install @coinbase/staking-client-library-ts
    ```
-2. Create and download an API key from the [Coinbase Developer Platform](https://portal.cdp.coinbase.com/access/api).
-3. Place the key named `.coinbase_cloud_api_key.json` at the root of this repository.
-4. Install necessary Typescript dependencies:
+
+2. Install necessary Typescript dependencies:
    ```shell
-    npm install -g ts-node
-    npm install -g typescript
-    ```
-5. Copy and paste one of the code samples below or any of our [provided examples](./examples/) into an `example.ts` file and run it with `ts-node` :rocket:
+   npm install -g ts-node typescript
+   ```
+
+3. Get your API keys info such as api key name and api private key from here: https://portal.cdp.coinbase.com/access/api. <br>
```

Comment:
I really like how the examples don't *assume* the user placed the API Key at a specific location anymore. It's best to be explicit in the code where possible. However, I do think it's a bit more cumbersome for a user to manually copy/paste the key's two fields into the example (including a private key, which isn't ideal from a security perspective). Especially since the CDP GUI downloads the key onto the filesystem already.

What if we asked the user to specify the file location of the API Key? So maybe it looks like this?

```ts
const apiKeyLocation: string = './.coinbase_cloud_api_key.json';

const client = new StakingClient(apiKeyLocation);
```

This is just an idea, so curious what you think.

Reply:
It depends what we are trying to do here - something I probably failed calling out in the PR description.. you're right downloading an API key or copy/pasting these 2 values feel equally difficult - it felt like exchanging one problem with perhaps a bigger one 😅 . But the primary aim of moving to coy/pasting these 2 details - key name and private key is so that:

1. We can move away from the coinbase_cloud api key file naming that the platform might take some time to update. We can skip the name confusion if we just copy/paste the relevant fields which the platform offers a good way of copy/pasting.

2. I believe the platform team is working on a token based auth model at which point in time, copy/pasting a token from platform UI will be the expected path. So this PR in a way preps for that eventual future. @deangalvin-cb can maybe confirm/deny this or add more color.

Reply:
👍 Yes, we are working directly with the https://github.com/coinbase/coinbase-sdk-ruby team to see if we can better align & unify our designs. I had @drohit-cb go this route as to not spend too much time building each iteration, and just making this quick improvement now.

Reply:
Kk I'm onboard 👍